charts/core/values/postgresql.yaml

```diff
@@ -6,11 +11,32 @@ primary:
 
         set -euo pipefail
 
-        ABSOLUTE_SEED_URL="https://databases.dhis2.org/sierra-leone/2.36.0/dhis2-db-sierra-leone.sql.gz"
-        curl -L "$ABSOLUTE_SEED_URL" -o /tmp/t$$ | cat
-        gunzip -c /tmp/t$$ > /tmp/t$$-seed-data
-        psql -U postgres -d dhis2 -p 5432 -f /tmp/t$$-seed-data
-        rm /tmp/t$$ /tmp/t$$-seed-data
+        export PGPASSWORD=$POSTGRES_POSTGRES_PASSWORD
+
+        function exec_psql() {
+          psql -U postgres -qAt -d dhis2 -c "$1"
+        }
+
+        table_exists=$(exec_psql "select exists (select from information_schema.tables where table_schema = 'schema_name' and table_name = 'table_name')")
+        if [[ $table_exists = true ]]; then
+          row_count=$(exec_psql "select count(*) from organisationunit")
+          if [[ $row_count -ne 0 ]]; then
+            echo "Seeding aborted!"
+            exit 0
+          fi
+        fi
+
+        SEED_URL="https://databases.dhis2.org/sierra-leone/2.39.0/dhis2-db-sierra-leone.sql.gz"
```

Comment:
Can this be configured? Just asking since the version of the DHIS2 image needs to be in sync with the DB dump

Reply:
No, it can not. But your comment made me realize something else. That file isn't part of the chart! It's only used by skaffold to configure the database. I've moved it to the root of the project, so it's next to the skaffold file and not in the chart folder.